.circleci/cimodel/data/pytorch_build_data.py

```diff
@@ -44,7 +44,7 @@
                     ("libtorch", [XImportant(True)])
                 ]),
             ]),
-            ("9.2", [X("3.6")]),
+            ("9.2", [XImportant("3.6")]),
```

Comment:
I promote the 9.2 test to be important so that we actually have a test running for the `torch.half` case of `test_blas_alpha_beta_empty`. I am OK with reverting this change, but if so, there will be no checks running on PR for this case.